src/app/header/header.component.html

```diff
@@ -0,0 +1,17 @@
+<md-toolbar [ngStyle]="{padding:0}">
```

Comment:
Please use css class

Reply:
Please add the theme to `style.css`: @import "~@angular/material/prebuilt-themes/indigo-pink.css";